### PR TITLE
Adds Bob's Name

### PR DESCRIPTION
**Pull request by smashdevcode**
This PR adds Bob's name to the list of names.

### CODE REVIEW

File: src/App.java [hunk 1]
```diff
@@ -9,6 +9,7 @@ public static void main(String[] args) {
         names.add("James");
         names.add("Kiel");
         names.add("Irina");
+        names.add("Bob");
```
Comment (kdonelson) [inline src/App.java]: @smashdevcode Ticket #45 said to add Fred's name, not Bob's. Could you please correct and resubmit

**Review by kdonelson — CHANGES_REQUESTED** [review]
@smashdevcode please extract the names ArrayList creation into its own method as well while you are correcting Bob's name to Fred. Thanks!

File: src/App.java (hunk 1 above)
Comment (smashdevcode) [reply src/App.java]: Oh no... working on this change now...

**Review by kdonelson — APPROVED** [review]
@smashdevcode Thanks for fixing that! :shipit:

**Pull request merged**